### PR TITLE
Update PR template and contribution guide

### PR DESCRIPTION
**Pull request by leightkt**
### Description
Updates PR template and Contribution Guide

Screenshot 2023-03-08 at 4 40 30 PM
Screenshot 2023-03-08 at 4 40 56 PM
Screenshot 2023-03-08 at 4 41 08 PM
Screenshot 2023-03-08 at 4 41 24 PM
Screenshot 2023-03-08 at 4 41 33 PM

### Checklist
- [ ] The version field in `package.json` of the package you have made changes in is incremented following [semantic versioning](https://semver.org/) and using alpha release tagging
- [ ] The box that allows repo maintainers to update this PR is checked
- [ ] I tested locally to make sure this feature/fix works
- [ ] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

### CODE REVIEW

File: docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md [hunk 1]
```diff
@@ -2,54 +2,74 @@
...
 [Core package.json](https://github.com/blocknative/web3-onboard/blob/8531a73d69365f7d584320f1c4b97a5d90f1c34e/packages/core/package.json#L3) changes: `"version": "2.9.1-alpha.1",` --> `"version": "2.9.1-alpha.2",`
 
+## [How can I add a new wallet? 💳](#new-wallet)
 
-### How can I get a new wallet added?
+To add a new wallet to the official Web3 Onboard packages and repo, create a [pull request](#pull-requests-🗂️). Web3 Onboard does not require a wallet to be a part of the main code, so a separate wallet module can be created without any changes to the Web3Onboard codebase. Your pull request must include a README for the package.  
 
-Web3 Onboard does not require a wallet to be a part of the main codebase to work, so a wallet module can be created and used for your project without needing anything to happen within the Web3Onboard codebase. If you would like the wallet to be part of the official Web3 Onboard packages and repo, then create a pull request, and make sure to add any documentation updates by creating a docs pull request.
+Please also add documentation updates by creating a separate [docs pull request](#documentation-contributions-📄).
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md:44]: The docs update should more often than not be part of the initial PR for wallet addition - the example PR linked below does have it separate but generally together is helpful.

File: docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md [hunk 2]
```diff
@@ -2,54 +2,74 @@
...
 Example : Adding a new injected wallet to the `injected` package -
 [Injected Package.json](https://github.com/blocknative/web3-onboard/blob/develop/packages/injected/package.json#L3) changes : `"version": "2.2.4",` --> `"version": "2.3.0-alpha.1",`
 
 Example 2 : Bug fix within the `core` package -
 [Core package.json](https://github.com/blocknative/web3-onboard/blob/8531a73d69365f7d584320f1c4b97a5d90f1c34e/packages/core/package.json#L3) changes: `"version": "2.9.1-alpha.1",` --> `"version": "2.9.1-alpha.2",`
 
+## [How can I add a new wallet? 💳](#new-wallet)
 
-### How can I get a new wallet added?
+To add a new wallet to the official Web3 Onboard packages and repo, create a [pull request](#pull-requests-🗂️). Web3 Onboard does not require a wallet to be a part of the main code, so a separate wallet module can be created without any changes to the Web3Onboard codebase. Your pull request must include a README for the package.
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md:42]: Might be good to add something like`quality README` and something around this being the first point of contact for dapp devs...wallet devs seem to put the smallest effort into documentation lol

File: pull_request_template.md [hunk 3]
```diff
@@ -2,11 +2,18 @@
...
 ### Checklist
-- [ ] The version field in `package.json` of the package you have made changes in is incremented following [semantic versioning](https://semver.org/) and using alpha release tagging
-- [ ] The box that allows repo maintainers to update this PR is checked
-- [ ] I tested locally to make sure this feature/fix works
-- [ ] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
-- [ ] This PR passes the Circle CI checks
+- [ ] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
+- [ ] Check the box that allows repo maintainers to update this PR
+- [ ] Test locally to make sure this feature/fix works
+- [ ] Run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
```
Comment (Adamj1232) [inline pull_request_template.md:8]:
```suggestion
- [ ] Run `yarn check-all` to confirm there are not any associated errors
```
Added a command that runs all 3 a few weeks back

File: docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md [hunk 4]
```diff
@@ -2,54 +2,74 @@
...
 
-Web3 Onboard does not require a wallet to be a part of the main codebase to work, so a wallet module can be created and used for your project without needing anything to happen within the Web3Onboard codebase. If you would like the wallet to be part of the official Web3 Onboard packages and repo, then create a pull request, and make sure to add any documentation updates by creating a docs pull request.
+Please also add documentation updates by creating a separate [docs pull request](#documentation-contributions-📄).
 
-If the wallet you are adding is an “injected” wallet (browser extension, mobile dapp browser wallet), you can add a wallet to the injected wallets module. [See here for an example of an injected wallets pull request.](https://github.com/blocknative/web3-onboard/pull/1177/files) You should also add the wallet to the [natively supported injected wallets list](https://onboard.blocknative.com/docs/packages/injected#injected-wallets-supported-natively).
+### Adding Injected Wallets
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md:46]: Might be helpful to reiterate that the PR checklist must be complete prior to review in a few spots throughout this

File: pull_request_template.md [hunk 5]
```diff
@@ -2,11 +2,18 @@
 <!-- Add a description of the fix or feature here -->
```
Comment (Adamj1232) [inline pull_request_template.md:2]: Might be helpful add something here around if the checklist related to your PR is not completed we cannot review

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Few changes but it looks a lot better! Thanks for doing this @leightkt!!

File: docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md (hunk 2 above)
Comment (leightkt) [reply docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md:42]: haha noted. I'll add some emphasis.

File: pull_request_template.md (hunk 3 above)
Comment (leightkt) [reply pull_request_template.md:8]: yessssss love the streamlining!

**Comment by leightkt** [comment]
Screenshot 2023-03-09 at 11 13 57 AM
Screenshot 2023-03-09 at 11 14 20 AM
Screenshot 2023-03-09 at 11 14 56 AM

**Review by Adamj1232 — APPROVED** [review]
Looks good!

**Review by aaronbarnardsound — COMMENTED** [review]
Looks good!

**Review by Adamj1232 — APPROVED** [review]

**Pull request merged**